source/includes/_run.md

```diff
@@ -1,30 +1,64 @@
-# Run irohad locally
+# Run the daemon (irohad) 
 
-## Generate genesis block
+## Create genesis block
 
-1. Create peers.list file containing the peers' addresses. For the local node suffice to add only single peer:
-```bash
-$ echo 'localhost:10001' > peers.list
-```
+A genesis block is the first block in the chain. <br>
+<b>Important:</b> each peer in the initialized network should have exactly the same genesis block, so create it once and distribute across peers. This block may have <b>any data</b>, as no-one validates it in the peer network.
+
+### Using iroha-cli
+
+#### Create list of peers
+Create peers.list file containing network addresses of peers' internal ports in the network (a single peer address goes on new line. If you want to run Iroha on single peer, it is sufficient to mention only localhost: <br> `$ echo 'localhost:10001' > peers.list`
+
+As an example, for 4 peers the file will look like the following:
+<ul>
+  <li>10.128.13.1:10001</li>
```

Comment:
As far as I remember first line in peers.list refers to ordering service (internal port), and has port number 10001. Remaining ones correspond to the peers' addresses and have port 50051

Reply:
okay...